src/Core/ServiceBuilder.php

```diff
@@ -317,12 +342,15 @@ public function trace(array $config = [])
      * ```
      *
      * @param array $config [optional] Configuration options. See
-     *        {@see Google\Cloud\ServiceBuilder::__construct()} for the available options.
+     *        {@see Google\Cloud\Core\ServiceBuilder::__construct()} for the available options.
      * @return VisionClient
      */
     public function vision(array $config = [])
     {
-        return new VisionClient($config ? $this->resolveConfig($config) : $this->config);
+        if (class_exists('Google\Cloud\Vision\VisionClient')) {
+            return new VisionClient($config ? $this->resolveConfig($config) : $this->config);
+        }
+        self::throwPackageNotFoundException('vision');
```

Comment:
Might not matter based on previous comment, but why call this in static context? The method isn't declated static.